WISE-Community/WISE-Client
# chore(Classroom Monitor): Convert ProjectProgress

## Changes

Converted ProjectProgress from AngularJS to Angular.

## Test

1. Open the Classroom Monitor for a run that some students have worked on
2. Go to the "Grade By Team" view
3. For each workgroup, look at the "Unit Completion" column and make sure the percentage bar works and the tooltip on the percentage bar works

Closes #786
